File: ds1/stack.py
def createStack():
    return []


def pushToStack(stack,value):
    return stack.append(value)


def popFromStack(stack):
    if not isStackEmpty(stack):
        stack.pop()
    else:
        print('stack is empty')
    

def peekStack(stack):
    if not isStackEmpty(stack):
        return stack[-1]
    else:
        print('stack is empty')
    


def isStackEmpty(stack):
    return len(stack) == 0


stack1 = createStack()
# pushToStack(stack1,'1')
# pushToStack(stack1,'1')
# pushToStack(stack1,'1')
# pushToStack(stack1,'1')
popFromStack(stack1)
print(stack1,isStackEmpty(stack1),peekStack(stack1))

File: ds1/stack.js
function createStack(){
    return []
}

function pushToStack(stack,value){
    return stack.push(value)
}

function popFromStack(stack,value){
    if(!isStackEmpty(stack)){
        stack.pop()
    }
}

function peekStack(stack){
    return stack[stack.length - 1]
}

function isStackEmpty(stack){
    return stack.length === 0
}

const stack1 = createStack()
// pushToStack(stack1,'1')
// pushToStack(stack1,'1')
// pushToStack(stack1,'1')
// pushToStack(stack1,'1')
popFromStack(stack1)
console.log(stack1,isStackEmpty(stack1),peekStack(stack1))